dist/images/ovnkube.sh

```diff
@@ -1140,7 +1140,7 @@ ovn-node() {
     --logfile-maxage=${ovnkube_logfile_maxage} \
     ${hybrid_overlay_flags} \
     ${disable_snat_multiple_gws_flag} \
-    ${disable_pkt_mtu_check_flag} \
+    ${disable_gateway_mtu_flag} \
```

Comment:
ovnkube-node does not care for this config right ? so can be removed IMO

Reply:
Done